src/examples/example_client.cpp

```diff
@@ -36,6 +36,9 @@ int main(int argc, char** argv)
     std::string endpoint = "opc.tcp://127.0.0.1:4841/freeopcua/server/";
     //std::string endpoint = "opc.tcp://localhost:53530/OPCUA/SimulationServer/";
```

Comment:
Can you add en example string with username and password to example.cpp? since this is the only documentation we have for now :-(